packages/components/src/components/hds/app-side-nav/index.hbs

```diff
@@ -19,7 +19,8 @@
       <div class="hds-app-side-nav__overlay" {{on "click" this.toggleMinimizedStatus}} />
       {{! template-lint-enable no-invalid-interactive}}
       <Hds::AppSideNav::ToggleButton
-        aria-label={{this.toggleButtonAriaLabel}}
+        aria-label="App navigation"
```

Comment:
Let me know if this label is ok or if it should be changed, moved, and/or exposed so consumers can override

Reply:
Hmm.. I think it should match what the h2 calls it for consistency - so "Application local navigation"

Reply:
in this case, I suggest we point to the heading to avoid duplicating content
```suggestion
        aria-labelledby="hds-app-side-nav-header"
```